pages/auth/setup.vue

```diff
@@ -54,6 +54,10 @@ export default {
 
     const current = route.query[SETUP] || 'admin';
 
+    const mcmEnabled = await store.dispatch('management/find', {
+      type: MANAGEMENT.FEATURE, id: 'multi-cluster-management', opt: { url: `/v1/${ MANAGEMENT.FEATURE }/multi-cluster-management` }
+    })?.spec?.value !== false;
```

Comment:
Value is null if the value is set to its default, which is false if you've turned off the flag at deployment time - I used:

```
    const mcmFlag = await store.dispatch('management/find', {
      type: MANAGEMENT.FEATURE, id: 'multi-cluster-management', opt: { url: `/v1/${ MANAGEMENT.FEATURE }/multi-cluster-management` }
    });
    let mcmEnabled = mcmFlag?.spec?.value || mcmFlag?.status?.default;

    // Default to true if we didn't get a boolean
    mcmEnabled = typeof mcmEnabled === 'boolean' ? mcmEnabled : true;
```